client/src/Components/Register/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+
+class Register extends Component {
+    render() {
+        return (
+            <header>
+                <div className="container-fluid">
+                    <div className="bg-img">
+                    </div>
+                    <div className="row-fluid">
+                        <div className="LogoInSignUp">
+                            <h1>Sport News</h1>
+                        </div>
+                        <div className="LogIn">
```

Comment:
lets use one style for className . I prefer '-'

Reply:
done